docs/how-to-guides/setup-NGINX-Ingress-Controller.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Setup NGINX Ingress Controller
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Rancher Desktop, which uses K3s under the hood in turn uses Traefik as the default ingress controller for your kubernetes cluster. However, there are unique use cases where NGINX may be required or preferred. Below steps show how to use NGINX Ingress controller for a sample deployment.
+
+### Steps
+
+1. Uncheck `Enable Traefik` from the `Kubernetes Settings` page to disable Traefik. You may need to exit and restart Rancher Desktop for the change to take effect.
+
+2. Deploy the NGINX ingress controller via `helm` or `kubectl`.
+
+<Tabs groupId="deployment-approach">
+  <TabItem value="helm" default>
+
+```
+helm upgrade --install ingress-nginx ingress-nginx \
+  --repo https://kubernetes.github.io/ingress-nginx \
+  --namespace ingress-nginx --create-namespace
+```
+
+  </TabItem>
+  <TabItem value="kubectl">
+
+```
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/controller-v1.1.2/deploy/static/provider/cloud/deploy.yaml
+```
+
+  </TabItem>
+</Tabs>
+
+3. Wait for the ingress pods to come up and running.
+
+```
+kubectl get pods --namespace=ingress-nginx
+```
+
+4. Create a sample deployment and the associated service.
+
+```
+kubectl create deployment demo --image=nginx --port=80
+kubectl expose deployment demo
+```
+
+5. Create an ingress resource. The following command uses a host that maps to localhost.
+
+```
+kubectl create ingress demo-localhost --class=nginx --rule=demo.localdev.me/*=demo:80
```

Comment:
When I tried this command using zsh it didn't work. The `*` character was the issue. Could you, instead, do something like...

```
kubectl create ingress demo-localhost --class=nginx --rule="demo.localdev.me/*=demo:80"
```

Quoting the value makes it work. Tested in bash and zsh.

Reply:
Updated the command per your suggestion. Tested in windows powershell, cmd and Github bash where you command works.